### PR TITLE
This PR #33 that I closed because of Issue #34

### PR DESCRIPTION
**Pull request by jpobeda**

### CODE REVIEW

File: lib/mactrack_functions.php [hunk 1]
```diff
@@ -764,7 +764,7 @@ function build_InterfacesTable(&$device, &$ifIndexes, $getLinkPorts = FALSE, $ge
 			$insert_vals .= ",";
 		}
 
-		$mac_address = @xform_mac_address($ifPhysAddress[$ifIndex]);
+		$mac_address = @xform_mac_address(isset($ifPhysAddress[$ifIndex]));
```
Comment (cigamit) [inline lib/mactrack_functions.php]: This one does not appear correct.  It should be:

`$mac_address = isset($ifPhysAddress[$ifIndex]) ? xform_mac_address($ifPhysAddress[$ifIndex]):'';`

Can you make that change?

**Comment by cigamit** [comment]
Make the one change and this is ready to go in Javier.

**Comment by jpobeda** [comment]
Sorry about that, don't know what I've done there, code in my current env was fine hahaha the evil copy & paste probably

**Comment by cigamit** [comment]
Okay, I'm going to merge this now.  Looks good.

**Pull request merged**